src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsintegrationapi/models/probationoffendersearch/Address.kt

```diff
@@ -3,7 +3,27 @@ package uk.gov.justice.digital.hmpps.hmppsintegrationapi.models.probationoffende
 import uk.gov.justice.digital.hmpps.hmppsintegrationapi.models.Address as IntegrationAPIAddress
 
 data class Address(
+  val addressNumber: String?,
+  val district: String?,
+  val buildingName: String?,
+  val county: String?,
+  val from: String?,
   val postcode: String?,
+  val streetName: String?,
+  val to: String?,
+  val town: String?,
 ) {
-  fun toAddress() = IntegrationAPIAddress(postcode = this.postcode)
+  fun toAddress() = IntegrationAPIAddress(
+    country = "England",
+    county = this.county,
+    endDate = this.to,
+    locality = this.district,
+    name = this.buildingName,
+    number = this.addressNumber,
+    postcode = this.postcode,
+    startDate = this.from,
+    street = this.streetName,
+    town = this.town,
+    type = "Type?",
```

Comment:
I think this should be restructured and refactored:

premise = this.addressNumber,
name = this.buildingName,
streetAddress = this.streetName,
addressLocality = this.district,
addressTown = this.town,
addressRegion = this.county,
addressCountry = "England"
postalCode = this.postcode,
startDate = this.from,
endDate = endDate = this.to,
additionalType = "Type?",